docs/faq/programming.md

```diff
@@ -0,0 +1,176 @@
+---
+sidebar_position: 2
+sidebar_label: Programming smart contracts
+---
+
+# Venom Blockchain FAQ: Programming smart contracts
+
+<details>
+<summary>
+I know how to develop Ethereum smart contracts. What is the difference with Venom Blockchain?
+</summary>
+
+Your experience with EVM-based blockchains will be useful for developing smart contracts on Venom Blockchain. But there are some differences and the main one is the contracts communication model. You can check [this](../build/development-guides/comparing-of-ethereum-vs-venom-architectures.md) article to know the difference.
+
+Moreover, Venom Blockchain uses Threaded Solidity language, which is a superset of Solidity. You can check its specification [here](https://github.com/tonlabs/TON-Solidity-Compiler/blob/master/API.md)
+
+</details>
+
+<details>
+<summary>
+Which tools can help me for developing on Venom?
+</summary>
+
+Check out [this](../build/quick-start-on-testnet.md) article. We have an awesome [repository](https://github.com/venom-blockchain/awesome-venom) as well.
+
+</details>
+
+<details>
+<summary>
+Does Venom Blockchain have some improvement proposals or some standards for example of fungible/non-fungible tokens?
```

Comment:
Does Venom Blockchain have any improvement proposals or standards, for example of fungible/non-fungible tokens?